internal/runner/rspec.go

```diff
@@ -14,6 +14,17 @@ import (
 // For now, Rspec provides rspec specific behaviour to execute
 // and report on tests in the Rspec framework.
 type Rspec struct {
+	TestCommand string
```

Comment:
We can store Rspec related config here (e.g. test command, retry command, etc) for all functions to access.